src/js/components/modals/AppModalComponent.jsx

```diff
@@ -112,7 +112,8 @@ var AppModalComponent = React.createClass({
     // All status below 300 are actually not an error
     if (status < 300) {
       this.onCreateApp();
-    } else if (status === 409) {
+    } else if (status === 409 && data.deployments != null) {
+      // a 409 error without deployments is a field conflict
```

Comment:
sadly so. 

I wasn't able to get the tests running on my machine at all, so in Travis we trust.